python/cuml/cuml/datasets/blobs.py

```diff
@@ -173,7 +172,8 @@ def make_blobs(
     # Set the default output type to "cupy". This will be ignored if the user
     # has set `cuml.global_settings.output_type`. Only necessary for array
     # generation methods that do not take an array as input
```

Comment:
comment needs updating

Reply:
What does the output type "array" mean? Does it mean "leave it as the type that it is" or will it convert to a cuml array?